### ISSUE TITLE
Source JAR does not contain resources

### ISSUE THREAD

**Issue by sebastic** [simonpoole/OpeningHoursParser#44]
The josm Debian package uses the source JAR for its build, but it lacks the resources:
```
wget https://jcenter.bintray.com/ch/poole/OpeningHoursParser/0.21.1/OpeningHoursParser-0.21.1-sources.jar -P /tmp
jar -tvf /tmp/OpeningHoursParser-0.21.1-sources.jar 
     0 Wed Mar 11 12:06:20 CET 2020 META-INF/
    25 Sun Mar 01 16:54:50 CET 2020 META-INF/MANIFEST.MF
     0 Fri Aug 26 15:45:54 CEST 2016 ch/
     0 Fri Aug 26 15:45:54 CEST 2016 ch/poole/
     0 Wed Mar 11 11:40:34 CET 2020 ch/poole/openinghoursparser/
   193 Wed May 08 23:45:04 CEST 2019 ch/poole/openinghoursparser/Copy.java
  5670 Sun Mar 01 13:12:18 CET 2020 ch/poole/openinghoursparser/DateRange.java
 11339 Sun Mar 01 13:12:18 CET 2020 ch/poole/openinghoursparser/DateWithOffset.java
   638 Wed May 08 23:42:54 CEST 2019 ch/poole/openinghoursparser/Element.java
  2600 Sun Mar 01 13:12:18 CET 2020 ch/poole/openinghoursparser/Event.java
  4477 Sat Feb 29 21:20:20 CET 2020 ch/poole/openinghoursparser/Holiday.java
  1228 Sun Mar 01 13:12:18 CET 2020 ch/poole/openinghoursparser/I18n.java
  3527 Sun Mar 01 13:12:18 CET 2020 ch/poole/openinghoursparser/Month.java
  3371 Wed May 08 23:57:44 CEST 2019 ch/poole/openinghoursparser/Nth.java
  3366 Mon Mar 09 21:14:02 CET 2020 ch/poole/openinghoursparser/OpeningHoursParseException.java
 29135 Wed Mar 11 11:40:34 CET 2020 ch/poole/openinghoursparser/OpeningHoursParser.jj
    97 Wed May 08 21:22:28 CEST 2019 ch/poole/openinghoursparser/package-info.java
 14397 Sat Feb 29 21:20:20 CET 2020 ch/poole/openinghoursparser/Rule.java
  5236 Sun Mar 01 13:12:18 CET 2020 ch/poole/openinghoursparser/RuleModifier.java
  6852 Sun Mar 01 13:12:18 CET 2020 ch/poole/openinghoursparser/TimeSpan.java
  7457 Sat Feb 29 21:20:20 CET 2020 ch/poole/openinghoursparser/Util.java
  2585 Sun Mar 01 13:12:18 CET 2020 ch/poole/openinghoursparser/VarDate.java
  3918 Sun Mar 01 13:12:18 CET 2020 ch/poole/openinghoursparser/VariableTime.java
  2616 Sun Mar 01 13:12:18 CET 2020 ch/poole/openinghoursparser/WeekDay.java
  6926 Sun Mar 01 13:12:18 CET 2020 ch/poole/openinghoursparser/WeekDayRange.java
  4510 Sun Mar 01 13:12:18 CET 2020 ch/poole/openinghoursparser/WeekRange.java
  4715 Sun Mar 01 13:12:18 CET 2020 ch/poole/openinghoursparser/YearRange.java
  8953 Sun Mar 01 16:04:58 CET 2020 ch/poole/openinghoursparser/DataTest.java
 21692 Mon Mar 02 16:05:24 CET 2020 ch/poole/openinghoursparser/UnitTest.java
```
This causes an exception (as reported by Andrew Harvey in [Debian Bug #956574](https://bugs.debian.org/956574)):
```
java.util.MissingResourceException: Can't find bundle for base name
ch.poole.openinghoursparser.Messages, locale
```
I'm not sure how `build.gradle` should be changed to also include the resources when building the source JAR.

**Comment by simonpoole** [comment]
@sebastic thanks for the report, the issue should be fixed in 0.21.3, please feel free to reopen if there is any further issue.

**Comment by sebastic** [comment]
Thanks for the quick fix, the source JAR looks good now.
